plugins/generic/dublinCoreMeta/DublinCoreMetaPlugin.php

```diff
@@ -115,6 +116,10 @@ public function monographView($hookName, $args)
             }
         }
 
+        if ($publication->getDoi()) {
```

Comment:
Sorry, maybe one more comment/change: could we here consider `isChapterRequest` and take the chapter DOI in that case -- similar to authors or abstracts above?

Reply:
Should I add it here, or follow that (https://github.com/pkp/pkp-lib/issues/7608#issuecomment-1305537725) and leave it for the mentioned issue?

Reply:
Yes, please. -- because we are adding that line, we could consider if chapter page, as it is for other metadata, and it is not a big change...

Reply:
@bozana there is this (https://github.com/pkp/omp/blob/main/classes/doi/Repository.php#L108) function that seems to get all the dois for a submission - what do you say, to use this function and add all of them - I think there is no distinction between different dois in DC, so using that will cover all, right?

Reply:
For now I changed it to 

```
$doi = $isChapterRequest ? $chapter->getDoi() : $publication->getDoi();
if ($doi) {
    $templateMgr->addHeader('dublinCorePubIdDOI', '<meta name="DC.Identifier.DOI" content="' . htmlspecialchars($doi) . '"/>');
}
```

Reply:
Hmmm... maybe for the moment just either the publication DOI or chapter DOI (if it is a chapter page) -- because we have different DC meta content for abstract page, chapter landing page and (monograph and chapter) file page, where we differentiate the metadata... and not to change the system we used till now at the moment... 
I will have to read a little bit about DC meta tags... but such changes I would leave for that other issue where we will improve the DCMetaPlugin...
Thanks a lot for the suggestion! :pray: